src/audiomixerboard.cpp

```diff
@@ -661,6 +662,58 @@ CAudioMixerBoard::CAudioMixerBoard ( QWidget* parent, Qt::WindowFlags ) :
     QObject::connect ( vecpChanFader[48], SIGNAL ( gainValueChanged ( double ) ), this, SLOT ( OnGainValueChangedCh48 ( double ) ) );
     QObject::connect ( vecpChanFader[49], SIGNAL ( gainValueChanged ( double ) ), this, SLOT ( OnGainValueChangedCh49 ( double ) ) );
 
+//Pans: ----
```

Comment:
Ups... Didn't you follow the latest code on master? I decided to remove Qt4 compatibility recently and removed all this code. So the actual code which is used is the Qt5 code which is much cleaner (no code repetitions).

Reply:
I just saw that you merged from master but this was an old version. Please merge again with the latest master code. (Note that the source code of Jamulus is heavily modified currently. So, unfortunately, it will require some merging to do on private branches...)